### ISSUE TITLE
Library specific tutorials are not easily discoverable

### ISSUE THREAD

**Issue by azeey** [gazebo-web/gazebosim-web-frontend#19]
Currently, to get to the gz-sim tutorials, one has to expand the "Library reference" menu, select Sim, and finally select Tutorials. I propose listing each library's tutorial in https://gazebosim.org/docs/garden/tutorials so they're just one link away from where users would go to find tutorials.

**Comment by j-rivero** [comment]
Before we start to add new tutorials, I would like to propose the reorder of the sections. See https://github.com/gazebosim/docs/pull/348

**Comment by mabelzhang** [comment]
A followup issue is that the per-library links in the https://gazebosim.org/libs page are broken. I'm not finding the source code to that page anywhere.

**Comment by mabelzhang** [comment]
I added the tutorial links as proposed in the original description, in a PR based on Jose's PR above https://github.com/gazebosim/docs/pull/349

**Comment by j-rivero** [comment]
As a side note: we can improve the accessibility of the tutorials for the users navigating the page but we might have a serious problem with web searches #20

**Comment by mabelzhang** [comment]
Does the page need to be deployed manually? It isn't up.

**Comment by azeey** [comment]
> Does the page need to be deployed manually? It isn't up.

Yes, gazebosim-backend has to be deployed manually each time we make changes to gazebosim/docs. I'll deploy it now and close this issue.

**Comment by mabelzhang** [comment]
Thanks! Now that it's deployed, I'm seeing some of the links I added are broken :sweat_smile:

The sdformat link is broken because it's slightly different... fixed in https://github.com/gazebosim/docs/pull/353

The gz-cmake and gz-utils links are broken. These specific ones are broken even if accessed from the Library Reference links - the API page is accessible, but the tutorial page doesn't exist.
https://gazebosim.org/api/cmake/3
https://gazebosim.org/api/utils/2
Should we just remove these from the newly added list, as opposed to having links that don't work?

**Comment by j-rivero** [comment]
> These specific ones are broken even if accessed from the Library Reference links - the API page is accessible, but the tutorial page doesn't exist.
> https://gazebosim.org/api/cmake/3
> https://gazebosim.org/api/utils/2
> Should we just remove these from the newly added list, as opposed to having links that don't work?

As one of the maintainers of cmake3 I think that we should try to add the tutorials or fix whatever is preventing them to have the tutorials page. Not that I think that removing them is the best temporary solution but given that we need to go win a cycle of manually deployment, if fix that in the code side of the libraries and that solves the problem best seems to me like going forward if possible.
